backends/arm/arm_backend.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Arm Limited and/or its affiliates.
+# Copyright 2023-2024 Arm Limited and/or its affiliates.
```

Comment:
Happy new year to you too :)

Reply:
thanks Digant! Yes, no functional changes. And it's ready to be merged. : )